assets/css/custom.css

```diff
@@ -48,6 +48,13 @@ $accessible-gray-column-color: $gray-column-color;
   background-color: $accessible-neutral-color;
 }
 
+.accessible-colors .navbar-default .navbar-nav>li>a{
+  color: #707070; /* Grey */
+}
+
+.accessible-colors{
```

Comment:
This is a bit of difficulty, as default accent color (`#f98012`) is a brand color. Anyway, a step in a good direction.